collectors/SDRSPropertiesCollector.py

```diff
@@ -0,0 +1,60 @@
+from collectors.PropertiesCollector import PropertiesCollector
+import json
+
+class SDRSPropertiesCollector(PropertiesCollector):
+
+    def __init__(self):
+        super().__init__()
+        self.vrops_entity_name = 'storagepod' 
+        self.label_names = [self.vrops_entity_name, 'vcenter', 'datacenter']
+        self.nested_value_metric_keys = [
+                "config|sdrsconfig|vmStorageAntiAffinityRules"
+                ]
+
+    def get_resource_uuids(self):
+        return self.get_SDRS_clusters_by_target()
+
+    def get_labels(self, resource_id, project_ids):
+        return [self.sdrs_clusters[resource_id]['name'],
+                self.sdrs_clusters[resource_id]['vcenter'],
+                self.sdrs_clusters[resource_id]['parent_dc_name'].lower()] if resource_id in self.sdrs_clusters else []
+
+    def unlock_nested_values(self, statkey, metric_value):
+
+        match statkey:
+            case "config|sdrsconfig|vmStorageAntiAffinityRules":
+                return self.config_sdrsconfig_vmStorageAntiAffinityRules(metric_value)
+
+    def config_sdrsconfig_vmStorageAntiAffinityRules(self, metric_value):
+
+        metric_value = json.loads(metric_value)
+        rules = metric_value.get("rules") if metric_value.get("rules") else []
+
+        amount_rules = len(rules) if rules else 0
```

Comment:
no need to have the if condition, len will return 0 if rules is []
```suggestion
        amount_rules = len(rules)
```